metricflow/dataflow/builder/node_evaluator.py

```diff
@@ -29,12 +29,12 @@
     ValidityWindowJoinDescription,
 )
 from metricflow.instances import InstanceSet
-from metricflow.model.semantics.semantic_model_join_evaluator import DataSourceJoinEvaluator
+from metricflow.model.semantics.semantic_model_join_evaluator import SemanticModelJoinEvaluator
```

Comment:
Pausing here (at the start of b08305294bc). So far so good. Will pick this back up in the morning.